README.md

```diff
@@ -29,3 +29,31 @@ Gitpod is provided as a [managed Saas version](https://gitpod.io) with a free su
 Open any GitLab, GitHub, or Bitbucket project, by prefixing the project's URL with `gitpod.io/#`. 
 
 E.g. https://gitpod.io/#https://github.com/gitpod-io/gitpod
+
+## Documentation
+
+All documentation can be found on https://docs.gitpod.io.
+For example, see [Introduction](https://www.gitpod.io/docs/) and [Getting Started](https://www.gitpod.io/docs/getting-started/) sections. 📚
+
+## Questions
+
+For questions and support please use the [community forum](http://community.gitpod.io/).
+Join the conversation, and connect with other community members. 💬
+
+You can also follow [`@gitpod`](https://twitter.com/gitpod) for announcements and updates from our team.
+
+## Issues
+
+The issue tracker is used for tracking **bug reports** and **feature requests** for the Gitpod open source project as well as planning current and future delopment efforts. 🗺️
```

Comment:
```suggestion
The issue tracker is used for tracking **bug reports** and **feature requests** for the Gitpod open source project as well as planning current and future development efforts.
``` 
:stuck_out_tongue:

Reply:
Thanks for taking a look @ArthurSens! Do you also suggest removing the 🗺️ emoji from there?

Reply:
Whoopis, no! that was an accident :sweat_smile: